learning_paths/models.py

```diff
@@ -1,3 +1,106 @@
 """
 Database models for learning_paths.
 """
+from django.db import models
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+class LearningPath(TimeStampedModel):
+    """
+    A Learning Path, containing a sequence of courses.
+
+    .. no_pii:
+    """
+
+    code = models.CharField(db_index=True, unique=True, max_length=255,
+                            verbose_name="Learning Path code",
+                            help_text="Custom unique code identifying this Learning Path.")
+    display_name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    # We don't use URLField here in order to allow e.g. relative URLs.
+    # max_length=200 as from URLField.
+    image_url = models.CharField(max_length=200, blank=True,
+                                 verbose_name="Image URL",
+                                 help_text="URL to an image representing this Learning Path.")
+    level = models.CharField(max_length=255, blank=True,
+                             help_text='E.g. "Beginner", "Intermediate", "Advanced".')
+    type = models.CharField(max_length=255, blank=True,
+                            verbose_name="Learning Path type",
+                            help_text='E.g. "Self-paced".')
```

Comment:
@Agrendalath, that's a very valid point. 

> Do we already have a clear use case for this field (e.g., filtering courses in the catalogs)? 

From the client's side, I don't think we have discussed "types of learning paths" explicitly as a requirement. So we don't actually "need" this field. If the course-discovery didn't consider it a crucial field, I would suggest removing it completely.

Maybe we can remove it and supply a default value in the API[^1]  for course-discovery to record. And if we end up adding the field as a requirement, the API can send the updated information. What do you think?

cc: @ArturGaspar 


[^1]: The API view currently doesn't exist. It will be created when we implement syncing of data between the plugin and course-discovery.